WebServiceManager/pom.xml

```diff
@@ -2,16 +2,16 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
     <modelVersion>4.0.0</modelVersion>
-    <groupId>com.raizlabs.net.webservicemanager</groupId>
+    <groupId>com.raizlabs.android.net.webservicemanager</groupId>
     <artifactId>RZWebServiceManager</artifactId>
     <packaging>apklib</packaging>
-    <name>RZWebServiceManager</name>
+    <name>RZWebServiceManager</name>

```

Comment:
#nth Are we going to go through with a rename at this point? If we're re-packaging, I would say we should do this simultaneously.